Bellman Equation

```diff
@@ -0,0 +1,268 @@
+\documentclass[12pt]{article}
+
+
+
+\addtolength{\textwidth}{.6in}
+
+\addtolength{\oddsidemargin}{-.3in}
+
+\setlength{\textheight}{8.4in}
+
+\setlength{\topmargin}{-.25in}
+
+\setlength{\headheight}{0.0in}
+
+\renewcommand{\baselinestretch}{1.0}
+
+\linespread{1.15}
+
+\pagenumbering{gobble}
+
+
+
+\usepackage{cite}
+
+\usepackage{times, verbatim,xcolor,bm}
+
+\usepackage{amsbsy,amssymb, amsmath, amsthm}
+
+\usepackage{hyperref}
+
+
+
+\newtheorem{definition}{Definition}
+
+\newtheorem{theorem}{Theorem}
+
+\newtheorem{lemma}{Lemma}
+
+\newtheorem{corollary}{Corollary}
+
+\newtheorem{assumption}{Assumption}
+
+\newtheorem{fact}{Fact}
+
+
+
+\newcommand{\ve}{\varepsilon}
+
+\newcommand{\ov}{\overline}
+
+\newcommand{\un}{\underline}
+
+\newcommand{\ta}{\theta}
+
+\newcommand{\Ta}{\Theta}
+
+\newcommand{\expect}{\mathbb{E}}
+
+
+
+\begin{document}
+
+\title{\vskip-0.6in Gradualism}
+
+\author{ECN 490}
+
+\date{May 1, 2018}
+
+\maketitle
+
+
+
+\section{Original Model with Government / Lobbying Effort}
+
+\title{Lobby}
+\begin{equation}
+\begin{aligned}
+\max_{e^t,m^t,l^t} \sum_{t=1}^\infty \beta^{t-1}\left\{A(m_t)F^\alpha\cdot l^{1-\alpha}_t\left[P^W+\tau(\gamma(e_{t-1}))\right]-l_t-\mu_t-e_t\right\}\\
+\text{s.t.} \hspace{5pt}  m_t=m_{t-1}+\mu_t\hskip.2in\\ 
+\end{aligned}
+\end{equation}
+
+\title{Bellman Equation}
+\begin{equation}
+\begin{aligned}
+V_l(e_t,m_t)=\max_{e_t,m_t,l_t}\left\{A(m_t)F^\alpha\cdot (l_t)^{1-\alpha}\left[P^W+\tau(\gamma(e_{t-1}))\right]-l_t-\mu_t-e_t+\beta V_l(e_{t+1},m_{m+t})\right\}\hskip.2in\\
+\end{aligned}
+\end{equation}
+
+\title{Value Function}
+\begin{equation}
+\begin{aligned}
+V_l(e_t^\ast,m_t^\ast)=\left\{A(m_t^\ast)F^\alpha\cdot (l^\ast_t)^{1-\alpha}\left[P^W+\tau(\gamma(e_{t-1}^\ast))\right]-l_t^\ast-\mu_t^\ast-e_t^\ast+\beta V_l(e^\ast_{t+1},m^\ast_{t+1})\right\} \hskip.2in\\
+\end{aligned}
+\end{equation}
+\[
+\]
+\normalsize
+Where $e_t$ denotes the initial state at time t, $m_t$ the level of technology at time t, $\mu_t$ the level of investment in technology in period t,  $\beta_t$ the discount factor, $\tau(\gamma(e_{t-1}))$ represent the tarif level as a function of the lobbying effort in the previous period. We use a Cobb-Douglas for the Lobby's utility function in which $A(m_t)$ determines the level of technology in the production process, $F^\alpha$ is the level of production, $l_t$ the lobbying effort at time t and $P^W$ is the world price. 
+\vskip.2in
+
+
+\section{No Government / Lobbying Effort, Technology Depreciates}
+
+\title{Lobby}
+
+\begin{equation}
+\begin{aligned}
+\max_{m^t,l^t} \sum_{t=1}^\infty \beta^{t-1}\left\{A(m_t)F^\alpha\cdot l^{1-\alpha}_t\cdot P^W-l_t-\mu_t\right\}\\ 
+\text{s.t.} \hskip.2in  m_{t+1}=(1-\delta)m_{t}+\mu_t\\
+\end{aligned}
+\end{equation}
+\[
+\]
+\normalsize
+By rearranging the restrictive condition and then replacing it in the Bellman Equation
```

Comment:
I would call m_{t+1}=(1-\delta)m_{t}+\mu_t the law of motion (there are several other terms for it, but I've never heard it called restrictive condition).